build-system/compile/bundles.config.extensions.json

```diff
@@ -1084,6 +1084,13 @@
       "hasCss": true
     }
   },
+  {
+    "name": "amp-story-subscription",
```

Comment:
All other subscriptions extensions use `subscriptions` plural, should we stick to that?

Reply:
Done.